modules/admin-ui-picard/app/src/components/users/partials/wizard/AclAccessPage.js

```diff
@@ -14,18 +14,22 @@ import {
 } from "../../../../thunks/aclThunks";
 import {addNotification} from "../../../../thunks/notificationThunks";
 import {removeNotificationWizardAccess} from "../../../../actions/notificationActions";
+import {getUserInformation} from "../../../../selectors/userInfoSelectors";
+import {hasAccess} from "../../../../utils/utils";
 
 /**
  * This component renders the access policy page in the new ACL wizard and in the ACL details modal
  */
-const AclAccessPage = ({ previousPage, nextPage, formik, addNotification, removeNotificationWizardAccess, isEdit }) => {
+const AclAccessPage = ({ previousPage, nextPage, formik, addNotification, removeNotificationWizardAccess, isEdit, user }) => {
     const { t } = useTranslation();
 
     const [aclTemplates, setAclTemplates] = useState([]);
     const [aclActions, setAclActions] = useState([]);
     const [roles, setRoles] = useState([]);
     const [loading, setLoading] = useState(false);
 
+    const isAccess = (isEdit && hasAccess("ROLE_UI_SERIES_DETAILS_ACL_EDIT", user)) || !isEdit;
```

Comment:
yes, because the `isEdit` flag shows if the current page is used in the details modal and can be edited or if the current page is used in a new resource wizard. Depending on this the page behaves slightly differently. If the page is used in the details modal the user also needs the role `ROLE_UI_SERIES_DETAILS_ACL_EDIT`. When the edit mode isn't considered for `isAccess` then the page could be messed up for new resources if the user also has the role `ROLE_UI_SERIES_DETAILS_ACL_EDIT`. In short, when used in a details modal the user also needs the role `ROLE_UI_SERIES_DETAILS_ACL_EDIT` to edit the ACLs and when used in a new resource modal the user doesn't need additional roles.